docs/reference/aggregations/reducer/movavg-aggregation.asciidoc

```diff
@@ -223,13 +223,108 @@ to see.  Small values emphasize long-term trends (such as a constant linear tren
 values emphasize short-term trends.  This will become more apparently when you are predicting values.
 
 [[double_0.2beta]]
-.Double Exponential moving average with window of size 100, alpha = 0.5, beta = 0.2
+.Holt-Linear moving average with window of size 100, alpha = 0.5, beta = 0.2
 image::images/reducers_movavg/double_0.2beta.png[]
 
 [[double_0.7beta]]
-.Double Exponential moving average with window of size 100, alpha = 0.5, beta = 0.7
+.Holt-Linear moving average with window of size 100, alpha = 0.5, beta = 0.7
 image::images/reducers_movavg/double_0.7beta.png[]
 
+==== Holt-Winters
+
+The `holt_winters` model (aka "triple exponential") incorporates a third exponential term which
+tracks the seasonal aspect of your data.  This aggregation therefore smooths based on three components: "level", "trend"
+and "seasonality".
+
+The level and trend calculation is identical to `ewma` The seasonal calculation looks at the difference between
```

Comment:
Shouldn't this be 'identical to the `Holt-Linear`'? isn't it `Holt-Linear` which smooths based on level and trend?